contracts/Liquadity/AuctionLiquadity.sol

```diff
@@ -521,7 +522,7 @@ contract Liquadity is Upgradeable,LiquadityFormula, TokenTransfer,InitializeInte
         baseToken = IERC20Token(_baseToken);
         mainToken = IERC20Token(_mainToken);
         relayToken = IERC20Token(_relayToken);
-        contractsRegistry = IAuctionRegistery(_registeryAddress);
+        
```

Comment:
We needed `contractsRegistry = IAuctionRegistery(_registeryAddress);` to be before `_updateAddresses()` is called.